fulp_modules/features/antagonists/bloodsuckers/code/structures/bloodsucker_crypt.dm

```diff
@@ -410,65 +407,84 @@
+	light_power = 1
+	light_range = 0
 	density = FALSE
 	anchored = FALSE
-	ghost_desc = "This is a magical candle which drains the sanity of non-Bloodsuckers and non-vassals.\n\
+	ghost_desc = "This magical light drains the sanity of non-Bloodsuckers and non-vassals.\n\
 		Only bloodsuckers and their vassals can toggle it."
-	vamp_desc = "This is a magical candle which drains at the sanity of unvassalized mortals while it is active.\n\
-		Only you and your vassals can toggle it."
-	vassal_desc = "This is a magical candle which drains at the sanity of the fools who havent yet accepted your master while active.\n\
+	vamp_desc = "This magical light drains the sanity of unvassalized mortals while active.\n\
+		Only you and your vassals can toggle it.\n\
+		You alone can also toggle it from afar by <b>double-clicking</b> it."
+	vassal_desc = "This is magical light drains the sanity of those fools who havent yet accepted your master while active.\n\
 		Only you, your master, and your master's other devotees may toggle it."
-	hunter_desc = "This is a blue candelabrum, which causes insanity to those near it while active."
+	hunter_desc = "This light causes insanity to those near it while active."
 	var/lit = FALSE
+	var/active_light_range = 3
 
-/obj/structure/bloodsucker/candelabrum/Destroy()
+/obj/structure/bloodsucker/lighting/Destroy()
+	. = ..()
 	STOP_PROCESSING(SSobj, src)
-	return ..()
 
-/obj/structure/bloodsucker/candelabrum/update_icon_state()
-	icon_state = "candelabrum[lit ? "_lit" : ""]"
-	return ..()
+/obj/structure/bloodsucker/lighting/update_icon_state()
+	. = ..()
+	icon_state = "[name][lit ? "_lit" : ""]"
 
-/obj/structure/bloodsucker/candelabrum/bolt()
+/obj/structure/bloodsucker/lighting/bolt()
 	. = ..()
 	set_anchored(TRUE)
 	density = TRUE
 
-/obj/structure/bloodsucker/candelabrum/unbolt()
+/obj/structure/bloodsucker/lighting/unbolt()
 	. = ..()
 	set_anchored(FALSE)
 	density = FALSE
+	if(lit)
+		toggle()
 
-/obj/structure/bloodsucker/candelabrum/attack_hand(mob/living/user, list/modifiers)
+/obj/structure/bloodsucker/lighting/attack_hand(mob/living/user, list/modifiers)
 	. = ..()
 	if(!.)
 		return
 	if(anchored && (IS_VASSAL(user) || IS_BLOODSUCKER(user)))
 		toggle()
 	return ..()
 
-/obj/structure/bloodsucker/candelabrum/proc/toggle(mob/user)
+/obj/structure/bloodsucker/lighting/DblClick()
+	. = ..()
+	if(in_range(src, usr))
+		return
+	if(anchored && IS_BLOODSUCKER(usr))
+		toggle()
+		usr.visible_message(span_danger("The [lit ? "[src.name] suddenly crackles to life" : "[src.name] is abruptly extinguished"]!"),
+		span_danger("<i>With a subtle hand motion you [lit ? "ignite [src]" : "snuff out [src]"].</i>"))
+
+/obj/structure/bloodsucker/lighting/proc/toggle(mob/user)
 	lit = !lit
 	if(lit)
 		desc = initial(desc)
-		set_light(2, 3, "#66FFFF")
+		set_light(active_light_range, light_power, light_color)
+		playsound(loc, 'sound/items/match_strike.ogg', 25)
+		if(name == "brazier")
+			particles = new /particles/brazier
```

Comment:
This should be on the brazier subtype instead, same for the check below this one

Reply:
Changed.